ledger-service/http-json/src/test/scala/com/digitalasset/http/HttpServiceTestFixture.scala

```diff
@@ -96,12 +97,15 @@ object HttpServiceTestFixture {
       a <- testFn(uri, encoder, decoder)
     } yield a
 
-    fa.onComplete { _ =>
-      ledgerF.foreach(_._1.close())
-      httpServiceF.foreach(_._1.unbind())
+    fa.transformWith { ta =>
+      Future
+        .sequence(
+          Seq(
+            ledgerF.map(_._1.close()),
+            httpServiceF.flatMap(_._1.unbind()),
+          ) map (_ fallbackTo Future.successful(())))
+        .flatMap(_ => Future fromTry ta)
```

Comment:
Did `ledgerF.foreach` or `httpServiceF.foreach` throw an exception?
I don't think the order is important... or is it?

Reply:
It didn't end up fixing the feature in this PR, but we were dropping errors on the floor, and `close` or `unbind` might not be finished when you started the next test, so I left it in.

Reply:
Well, I mean, we're still dropping errors on the floor, just in a deterministic way :)